node/resolvers/session/sessionResolver.ts

```diff
@@ -64,8 +105,54 @@ export const sessionFields = (session: any): SessionFields | {} => {
         impersonate: {
           ...setProfileData(namespaces.profile, namespaces.impersonate),
         },
+        utmParams: setUtmParams(namespaces.public),
+        utmiParams: setUtmiParams(namespaces.public),
         orderFormId: path(['public', 'orderFormId', 'value'], namespaces),
         ...setProfileData(namespaces.profile, namespaces.authentication),
       }
     : {}
 }
+
+export interface Session {
```

Comment:
You can create a Session.ts file inside the typings folder (just like the Catalog.ts) and put all related Session types there

Reply:
Done!